spec/models/report_spec.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Report, type: :model do
+  describe '#editable' do
+    let(:user) { User.create!(**params) }
+    let(:params) do
+      {
+        name: 'alice',
+        email: 'alice@example.com',
+        password: '123456',
+        password_confirmation: '123456'
+      }
+    end
```

Comment:
```suggestion
    let(:user) do 
      User.create!(
        name: 'alice',
        email: 'alice@example.com',
        password: '123456',
        password_confirmation: '123456'
      )
    end
```

paramsを別のletに分ける必要はないですね